Area51/Sources/Extensions/String+Extension.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+extension String {
+    var stripHtml: String {
```

Comment:
nit, since it returns a new string, should be called `strippingHTML`

Reply:
Generally, functions aren't named by what they're doing, more towards their post-actions or action-to-be-done. 

An example of this is in the Swift Standard library - for `Float` and `Double` - which have two functions; [round()](https://developer.apple.com/documentation/swift/float/2884011-round) and [rounded()](https://developer.apple.com/documentation/swift/float/2885331-rounded)

Using the above example, I feel `stripHtml` is just as appropriate as `strippedHtml` however I don't feel `strippingHtml` is the right choice here. 

I agree though, it should have a different name, since it returns a new string - if we follow the Standard library example, `strippedHtml` would seem more appropriate to me. 

I'll make this change before merging. 

I am however open to discussing this more though, off this PR. Maybe I am missing something more obvious.

Reply:
Either one sounds good